test/packing/log_auto_test.h

```diff
@@ -0,0 +1,1465 @@
+/* DO NOT EDIT: automatically built by log_auto-test-gen.sh */
+
+#pragma GCC diagnostic error "-Wno-used-but-marked-unused"
+#pragma clang diagnostic ignored "-Wused-but-marked-unused"
+
+#include "log_auto_fmt.h"
+#include "log_auto_direct.h"
+
+#define testutil_assertfmt__(a, fmt, ...)                                                         \
+    do {                                                                                          \
+        if (!(a))                                                                                 \
+            testutil_die(0, "\n%s:%d:\n  %s:\n    " fmt, __FILE__, __LINE__, __PRETTY_FUNCTION__, \
+              __VA_ARGS__);                                                                       \
+    } while (0)
+
+#define assert_eq(a, b) testutil_assertfmt__((a) == (b), "%s != %s", #a, #b)
+#define assert_eq_uint16_t(a, b) \
+    testutil_assertfmt__((a) == (b), "%s != %s : (%u) != (%u)", #a, #b, (a), (b))
+#define assert_eq_uint32_t(a, b) \
+    testutil_assertfmt__((a) == (b), "%s != %s : (%lu) != (%lu)", #a, #b, (a), (b))
+#define assert_eq_uint64_t(a, b) \
+    testutil_assertfmt__((a) == (b), "%s != %s : (%llu) != (%llu)", #a, #b, (a), (b))
+
+WT_ATTRIBUTE_UNUSED
+static void
+test_cmp_one__wt_logop_col_modify(WT_SESSION_IMPL *session, WT_ITEM *logrec_fmt,
+  WT_ITEM *logrec_direct, uint32_t fileid, uint64_t recno, WT_ITEM *value)
+{
+    const uint8_t *data_fmt, *data_direct;
+    /*size_t sz_fmt, sz_direct;*/
+    uint32_t fileid_fmt, fileid_direct;
+    uint64_t recno_fmt, recno_direct;
+    WT_ITEM value_fmt, value_direct;
+
+    fileid_fmt = fileid;
+    fileid_direct = fileid;
+    recno_fmt = recno;
+    recno_direct = recno;
+    value_fmt.data = value->data;
+    value_direct.data = value->data;
+    value_fmt.size = value->size;
+    value_direct.size = value->size;
+    logrec_fmt->size = logrec_direct->size = 0;
+    /*sz_fmt = logrec_fmt->size;*/
+    /*sz_direct = logrec_direct->size;*/
+
+    /* test packing */
```

Comment:
Didn't go over the entirety of this file either, but a couple of things:
* Some commented-out code
* A couple of comments that don't align with the style guide, i.e. start with uppercase and end with a full stop
* Would it be possible to reuse some existing test infra like `testutil_assert`?